### ISSUE TITLE
Need API for payments via Android on my apps

### ISSUE THREAD

**Issue by aishik212** [solana-labs/solana-pay#112]
I want to integrate solana pay on my apps, how can i do that?
And if possible these are the following i would prefer

1) Connect an users wallet
2)Add money via systems like razorpay
2.1) With the help of razorpay it'll be transferred to some wallet (this is my job) then it'll be transferred to the persons wallet(this will be done via api)
3) Then the person can use it from within my app as a money transfer service.

In short - paypal of Crypto world with bare minimum fees

**Comment by jordansexton** [comment]
We don't provide developer support or answer general questions on Github 🙂

Please ask in the #solana-pay channel on Discord: https://discord.com/invite/solana
